specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1086 @@
+          "description": "The query string used by Bing to perform the query. Bing uses the altered query string if the original query string contained spelling mistakes. For example, if the query string is \"saling downwind\", the altered query string will be \"sailing downwind\". This field is included only if the original query string contains a spelling mistake.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "alterationOverrideQuery": {
+          "description": "The query string to use to force Bing to use the original string. For example, if the query string is \"saling downwind\", the override query string will be \"+saling downwind\". Remember to encode the query string which results in \"%2Bsaling+downwind\". This field is included only if the original query string contains a spelling mistake.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "adultIntent": {
+          "description": "A Boolean value that indicates whether the specified query has adult intent. The value is true if the query has adult intent; otherwise, false.",
+          "readOnly": true,
+          "type": "boolean"
+        },
+        "askUserForLocation": {
+          "description": "A Boolean value that indicates whether Bing requires the user's location to provide accurate results. If you specified the user's location by using the X-MSEdge-ClientIP and X-Search-Location headers, you can ignore this field. For location aware queries, such as \"today's weather\" or \"restaurants near me\" that need the user's location to provide accurate results, this field is set to true. For location aware queries that include the location (for example, \"Seattle weather\"), this field is set to false. This field is also set to false for queries that are not location aware, such as \"best sellers\".",
+          "readOnly": true,
+          "type": "boolean"
+        }
+      }
+    },
+    "WebWebAnswer": {
+      "description": "Defines a list of relevant webpage links.",
+      "x-ms-discriminator-value": "Web/WebAnswer",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchResultsAnswer"
+        }
+      ],
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "description": "A list of webpages that are relevant to the query.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WebPage"
+          }
+        },
+        "someResultsRemoved": {
+          "description": "A Boolean value that indicates whether the response excluded some results from the answer. If Bing excluded some results, the value is true.",
+          "readOnly": true,
+          "type": "boolean"
+        }
+      }
+    },
+    "Images": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchResultsAnswer"
+        }
+      ],
+      "type": "object",
+      "required": [
+        "value"
```

Comment:
Looks like value isn't one of the properties below, and is causing this failure:

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/286652039#L579

Reply:
Looking into this

Reply:
Fixed this